plugins/woocommerce/src/StoreApi/Utilities/OrderController.php

```diff
@@ -289,10 +343,10 @@ protected function validate_email( \WC_Order $order ) {
 	 *
 	 * @throws RouteException Exception if invalid data is detected.
 	 * @param \WC_Order $order Order object.
+	 * @param bool      $needs_shipping Whether the order needs shipping.
 	 */
-	protected function validate_addresses( \WC_Order $order ) {
+	protected function validate_addresses( \WC_Order $order, bool $needs_shipping ) {
```

Comment:
Since this is a protected method should this be classified as a breaking (major) change? Or is it fine to keep it as a minor change?

Reply:
no I think this is fine